incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -50,41 +58,52 @@
 import io.aklivity.zilla.runtime.command.dump.internal.types.IPv6HeaderFW;
-import io.aklivity.zilla.runtime.command.dump.internal.types.OctetsFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.PcapGlobalHeaderFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.PcapPacketHeaderFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.TcpFlag;
 import io.aklivity.zilla.runtime.command.dump.internal.types.TcpHeaderFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.AbortFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.BeginFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.ChallengeFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.DataFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.EndFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.ExtensionFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.FlushFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.FrameFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.ResetFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.SignalFW;
 import io.aklivity.zilla.runtime.command.dump.internal.types.stream.WindowFW;
 import io.aklivity.zilla.runtime.engine.binding.function.MessagePredicate;
+import io.aklivity.zilla.runtime.engine.config.KindConfig;
+import io.aklivity.zilla.runtime.engine.reader.BindingsLayoutReader;
 
 @Command(name = "dump", description = "Dump stream content")
 public final class ZillaDumpCommand extends ZillaCommand
 {
+    private static final String OPTION_PROPERTIES_PATH_DEFAULT = ".zilla/zilla.properties";
     private static final Pattern PATTERN_NAMESPACED_BINDING = Pattern.compile("(?<namespace>[^\\.]+)\\.(?<binding>[^\\\\.]+)");
+    private static final Pattern STREAMS_PATTERN = Pattern.compile("data(\\d+)");
 
     private static final long MAX_PARK_NS = MILLISECONDS.toNanos(100L);
     private static final long MIN_PARK_NS = MILLISECONDS.toNanos(1L);
     private static final int MAX_YIELDS = 30;
     private static final int MAX_SPINS = 20;
     private static final int BUFFER_SLOT_CAPACITY = 64 * 1024;
+    private static final int LABELS_BUFFER_SLOT_CAPACITY = 4 * 128;
 
     private static final long PCAP_GLOBAL_MAGIC = 2712847316L;
     private static final short PCAP_GLOBAL_VERSION_MAJOR = 2;
     private static final short PCAP_GLOBAL_VERSION_MINOR = 4;
-
     private static final int PCAP_GLOBAL_SIZE = 24;
-
     private static final int PCAP_LINK_TYPE_IPV6 = 1;
 
+    private static final byte[] PSEUDO_ETHERNET_FRAME = BitUtil.fromHex("2052454356002053454e440086dd");
+    private static final int PSEUDO_IPV6_PREFIX = 1629561669;
+    private static final short PSEUDO_NEXT_HEADER_AND_HOP_LIMIT = 0x0640;
+    private static final int IPV6_LOCAL_ADDRESS = 0xfe80;
```

Comment:
```suggestion
    private static final long IPV6_LOCAL_ADDRESS = 0xfe80L << 48;
```

The `fe80` should be on the high bits.
```
jshell> String.format("%x", 0xfe80L << 48)
$2 ==> "fe80000000000000"
```

Reply:
OK